docs/src/index.md

```diff
@@ -4,6 +4,39 @@ CurrentModule = Survey
+
+# Why a package for survey analysis in Julia?
+
+## Julia is better in many classes of problems
+Several software tools are available to analyse complex surveys[^list_packages]. The R [survey](https://r-survey.r-forge.r-project.org/survey/) package is a widely used open-source option, but while it excels in expressivity, it's not very performant as it's completely written in R. Generating summary statistics on really large surveys sometimes takes hours. Analytical taylor-series based methods for variances are reasonably fast, but replicate weights based methods are painfully slow. A great number of surveys these days provide replicate weights instead of probability weights for statistical disclosure control purposes. In this class of simulation problems, Julia has a substantial edge over R.
+## Advantages of Julia `Survey.jl`
+1. There has been interest in a package for survey analysis in the Julia community for quite some time[^2], and some attempts to create a package, which never materialised[^3]. 
+2. Julia is able to combine the **expressivity** of R/Python with the **speed** of a systems programming language, so it is ideally placed to be the foundation for a survey analysis framework. 
+3. Avoids two-language problem. Fast data frameworks in R or Python usually rely on behind the scenes calls to C, C++, Fortran for speedups, as native interpreted code is usually slow. R `survey` is written in pure R and ease of use, not efficiency, was the main development goal. While there are addons and extensions, a completely Julia-native survey analysis package is great for users as well as long term development and maintenance.
+4. The statistical and data ecosytem of Julia has also matured to have substantial statistical computing abilities. `DataFrames.jl` provides the excellent data wrangling and "split-apply-combine" style analytics widely used survey analysis workflows[^dataframes.jl]. `Makie` provides a tightly integrated visualisation backend for graphical analysis of surveys[^makie]. `LinearAlgebra` and `Optim` as well as auto-differentiation libraries provide cutting edge foundations for further statistical modelling and analysis.
+
+# Future plans
```

Comment:
A 'future plans' a phrase or it's just 'plans'?

Reply:
changed